src/uiTest/resources/project.rhino.js

```diff
@@ -22,15 +22,32 @@ importClass(org.assertj.swing.fixture.JTableFixture);
 importClass(org.assertj.swing.data.TableCell);
 
 
+let myClick = function (component, mouseButton) {
```

Comment:
btw, for UITestSuite there is no need for "pretty click". What more it's even better to speed up the process and click the buttons ASAP. That will save us some time too